docs/E2ETESTING.md

```diff
@@ -0,0 +1,77 @@
+# End-to-End (E2E) Testing with Stark
+
+The stark starter comes with a setup to run e2e tests with protractor.
+
+The showcase app uses Travis CI and BrowserStack to automatically run e2e-tests on multiple setups (IE, Safari, Android, IOS, ...).
+
+## Protractor
+
+Protractor is an end-to-end test framework for Angular and AngularJS applications. Protractor runs tests against your application running in a real browser, interacting with it as a user would.
+
+For more information go to [https://www.protractortest.org](https://www.protractortest.org).
+
+## BrowserStack (optional)
+
+If you want to run your e2e-tests automatically on multiple setups one of the easiest solutions is BrowserStack.
+They offer a free solution for open source projects.
+
+For more information go to [https://www.browserstack.com/automate](https://www.browserstack.com/automate).
+
+## Running end-to-end tests
+
+Install all dependencies.
+
+```shell
+npm run install:all
+```
+
+Build the application
+
+```shell
+npm run build:showcase
```

Comment:
I think this command should refer to the Starter since we are talking about it at the beginning of the doc.

```
npm run build:starter
```